Makefile

```diff
@@ -28,6 +28,9 @@ build-all:
 	$(MAKE) build-docker GOOS=darwin GOARCH=amd64
 	$(MAKE) build-docker GOOS=linux GOARCH=amd64
 
+build-all-latest:
+	$(MAKE) build-all VERSION=latest
```

Comment:
The examples I gave in my earlier comment are actually what you want (I know, make syntax is weird):

```makefile
build-all-latest: VERSION=latest
build-all-latest: build-all
```